coordinator/points_writer_test.go

```diff
@@ -145,7 +222,7 @@ func TestPointsWriter_MapShards_Invalid(t *testing.T) {
 	}
 
 	// Add a point that goes beyond the current retention policy.
-	pr.AddPoint("cpu", 1.0, time.Now().Add(-2*time.Hour), nil)
+	pr.AddPoint("cpu", 1.0, time.Now().Add(time.Hour), nil)
```

Comment:
To further clarify: the old code this is replacing would just assume that the shardgroup returned by "CreateShardGroup" was the correct one. The code now validates that the point _should_ be written to the returned shardgroup.